### DISCUSSION TITLE
dracut: FATAL: Don't know how to handle 'root=/zroot/ROOT/fedora'

### DISCUSSION THREAD

**Discussion by FireTrack** [zbm-dev/zfsbootmenu#564] [Questions] [answered]
### ZFSBootMenu build source

Release EFI

### ZFSBootMenu version

2.3.x

### Boot environment distribution

Fedora Workstation 39 - 1.5

### Problem description

Today I looking into installing ZFSbootmenu with Fedora 39.

Following the installation instructions at https://docs.zfsbootmenu.org/en/v2.3.x/guides/fedora/uefi.html and then trying to boot from the created zpool I receive the following error message:

dracut: FATAL: Don't know how to handle 'root=ZFS=/ROOT/fedora' 

The documentation seems to be missing a step at the end to create a new initramfs:

`# dracut --regenerate-all`

### Steps to reproduce

1) Follow install instructions at https://docs.zfsbootmenu.org/en/v2.3.x/guides/fedora/uefi.html
2) System will not be able to boot from new zpool

**Comment by zdykstra** [comment]
Both of the error messages you've reported involve a kernel command line that is impossible for ZFSBootMenu to generate. Just to confirm, your Fedora system boots correctly after you run `dracut --generate-all` in a chroot? If so, this is not a ZFSBootMenu bug but instead is a documentation issue.

**Reply by zdykstra** [reply, marked as answer]
Documentation has been updated to include an initramfs regeneration step for Fedora Workstation 39 - https://docs.zfsbootmenu.org/en/v2.3.x/guides/fedora/uefi.html#regenerate-initramfs
